### ISSUE TITLE
QuickCheck squorum_eqc_tests:prop_squorum1 fails with quorum_error

### ISSUE THREAD

**Issue by tatsuya6502** [hibari/gdss-admin#4]
Since: unknown, this test was already failing in Hibari v0.1.10

With this test setting, it seems `squorum:set/6` won't succeed on a cluster having more than two nodes.

```erl
% make eqc
...

prop_squorum1: 
...
[{set,{var,1},
      {call,squorum_eqc_tests,set_num_bricks,
            [{var,env_bricks},{var,env_reordered}]}},
 {set,{var,4},
      {call,squorum_eqc_tests,squorum_set,
            [{var,env_bricks},
             <<"/foo/bar/0001">>,<<>>,1363736647469185004,[]]}}]
Env = [{env_reordered,[{squorum_test1,gdss_eunit@localhost},
                       {squorum_test2,gdss_eunit@localhost},
                       {squorum_test3,gdss_eunit@localhost}]},
       {env_bricks,[{squorum_test1,gdss_eunit@localhost},
                    {squorum_test2,gdss_eunit@localhost},
                    {squorum_test3,gdss_eunit@localhost}]}]
S = {state,2,
           [{<<"/foo/bar/0001">>,[]},
            {<<"/foo/bar/0002">>,[]},
            {<<"/foo/bar/0003">>,[]},
            {<<"/foo/bar/0004">>,[]}],
           3,
           [{squorum_test1,gdss_eunit@localhost},
            {squorum_test2,gdss_eunit@localhost},
            {squorum_test3,gdss_eunit@localhost}],
           [{squorum_test1,gdss_eunit@localhost},
            {squorum_test2,gdss_eunit@localhost},
            {squorum_test3,gdss_eunit@localhost}]}
R = {postcondition,
        {'EXIT',
            {{case_clause,quorum_error},
             [{squorum_eqc_tests,postcondition,3,
                  [{file,"test/eqc/squorum_eqc_tests.erl"},{line,219}]},
              {eqc_statem,f587_0,5,[]},
              {eqc_statem,f574_0,5,[]},
              {eqc_statem,run_commands,3,[]},
              {squorum_eqc_tests,'-common1_prop/2-fun-2-',3,
                  [{file,"test/eqc/squorum_eqc_tests.erl"},{line,106}]},
              {eqc,'-f779_0/2-fun-4-',3,[]},
              {eqc_gen,'-f317_0/2-fun-0-',5,[]},
              {eqc_gen,f180_1,2,[]}]}}}
```

**Comment by tatsuya6502** [comment]
After replacing `gmt_eqc` wrapper with `qc` wrapper (or upgrading rebar), test cluster wasn't correctly bootstrapped in `prop_squorum1`. Fixed this by commit https://github.com/hibari/gdss-admin/commit/44ac9b38f0984ea2dfe7348c033b6d722b72d6c5

**Comment by tatsuya6502** [comment]
From the [first comment](https://github.com/hibari/gdss-admin/issues/4#issue-12208211):

> Since: unknown, this test was already failing in Hibari v0.1.10
> 
> With this test setting, it seems `squorum:set/6` won't succeed on a cluster having more than two nodes.
> 
> ```erl
> prop_squorum1: 
> ...
> R = {postcondition,
>         {'EXIT',
>             {{case_clause,quorum_error},
```

After committing this fix, `quorum_error` disappeared.
https://github.com/hibari/gdss-client/issues/2#issuecomment-85498236

**Comment by tatsuya6502** [comment]
Closing this issue as it no longer gets `quorum_error`.

**Comment by Sacrifiser** [comment]
2 ADMIN_NODES is not supported

Can some one write how to fix this issue ? 0.1.11
